core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -264,7 +264,10 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
    * */
   def unregisterBillingProjectWithOwnerInfo(projectName: RawlsBillingProjectName, ownerInfo: Map[String, String]): Future[PerRequestMessage] = {
     val ownerUserInfo = UserInfo(RawlsUserEmail(ownerInfo("newOwnerEmail")), OAuth2BearerToken(ownerInfo("newOwnerToken")), 3600, RawlsUserSubjectId("0"))
-    unregisterBillingProjectWithUserInfo(projectName, ownerUserInfo)
+    for {
+      _ <- samDAO.deleteResource(SamResourceTypeNames.googleProject, projectName.value, ownerUserInfo)
```

Comment:
I haven't looked at the test failures at all so take this with a grain of salt, but they _could_ be caused by undeleted pet service accounts. I think gpalloc cleans up all pets in a project when it reclaims it anyways so may not matter, but could be something to look at

Reply:
It was a flake. Wasn't actually a test failure, but the other job. I think maybe there was a timeout fetching a dependency or something.